src/main/java/com/gluonhq/substrate/model/ReleaseConfiguration.java

```diff
@@ -203,21 +207,29 @@ public String getPackageType() {
     }
 
     public String getAppDescription() {
-        return appDescription;
+        return isNullOrBlank(appDescription) ? DEFAULT_APP_DESCRIPTION : appDescription;
     }
 
     public void setAppDescription(String appDescription) {
         this.appDescription = appDescription;
     }
 
     public String getVendor() {
-        return vendor;
+        return isNullOrBlank(vendor) ? DEFAULT_VENDOR : vendor;
     }
 
     public void setVendor(String vendor) {
         this.vendor = vendor;
     }
 
+    public String getVersion() {
+        return Objects.requireNonNullElse(version, DEFAULT_VERSION);
```

Comment:
`isNullOrBlank` is a utility method which returns a boolean depending on the supplied string, where as, `requireNonNullElse` returns a String.

Reply:
I can see that, but then you use that boolean to choose the passed string if it is correct, or a default value, which is exactly what `requireNonNullElse` does... so unless you want to avoid the use of empty strings, those are the same?

Reply:
Yes, I want to avoid empty strings as well. Wix doesn't play well with empty strings.